### DISCUSSION TITLE
Poor fit of absorption of a sustained-release preparation

### DISCUSSION THREAD

**Discussion by Xiaoyang-svg** [Open-Systems-Pharmacology/Forum#1808] [Questions & Problems]
Dear community,

I am developing a PBPK model for a sustained-release formulation of a BCS-1 drug. In this model , I import the dissolution data of the formulation into the Formulation building block and validate this model using PK data from the same literature. However, the simulation results are not ideal (figure below).

<img width="375" alt="mmexport1722239141784" src="https://github.com/user-attachments/assets/02d0c560-e86b-4feb-9cd1-3e0fec501a72">

I then tried to optimize the parameters for dissolution shape and dissolution time (50% dissolved), but with no success. It seems that parameter optimization is difficult to fit both peak concentration and time to peak concentration. So I would like to know how to adjust the model to get a good fit of absorption or are there other extended release tutorials I can refer to?

Many thanks.

**Comment by AndreDlm** [comment]
Depending on the drug you are modeling and the information you have from, e.g., in vitro assays, you might consider fitting the intestinal permeability as well. Be cautious to avoid non-identifiability issues. If you have in vitro dissolution data of your formulation in biorelevant media, you could use an in vitro model and fit a dissolution function to these data first, before fixing them to the fitted values in your PBPK model
